src/main/java/TaskList.java

```diff
@@ -0,0 +1,40 @@
+import java.util.ArrayList;
+
+public class TaskList {
+    private static final ArrayList<Task> TaskList = new ArrayList<>(10);
+    private static int NumTasks = 0;
+    public static void addItem (Task newTask) {
+        TaskList.add(newTask);
+        NumTasks += 1;
+        System.out.println("Okay! I've added: [" + newTask.getTypeIcon() +"] " + newTask.getDescription());
+    }
+    public static int getNumItems() {
+        return NumTasks;
+    }
+    public static void viewList () {
+        for (int i = 0; i < TaskList.size(); ++i) {
+            System.out.print(i+1 + ". ");
+
+            System.out.println(TaskList.get(i).getTask());
+        }
+    }
+    public static void markDone (int index) {
```

Comment:
There should not be a space between `markDone` and `(`. Do change this for your other methods as well :)